stored_requests/events/api/api_test.go

```diff
@@ -16,15 +16,17 @@ import (
 
 func TestGoodRequests(t *testing.T) {
 	cache := stored_requests.Cache{
-		Requests: memory.NewCache(256*1024, -1, "Request"),
-		Imps:     memory.NewCache(256*1024, -1, "Imp"),
-		Accounts: memory.NewCache(256*1024, -1, "Account"),
+		Requests:  memory.NewCache(256*1024, -1, "Request"),
+		Imps:      memory.NewCache(256*1024, -1, "Imp"),
+		Responses: memory.NewCache(256*1024, -1, "Responseds"),
```

Comment:
Super nitpick: typo should be `"Responses"`

Reply:
Ooof, modified!